test/src/main/java/mil/nga/giat/geowave/test/service/ServicesTestEnvironment.java

```diff
@@ -69,6 +70,13 @@ protected static void writeConfigFile(
 			writer.println("instance=" + accumuloInstance);
 			writer.println("user=" + accumuloUser);
 			writer.println("password=" + accumuloPassword);
+			writer.println("gwNamespace=" + TEST_NAMESPACE);
```

Comment:
@datasedai here's another example of a quick/dirty workaround - the info service is using something closer to the old style of configuration parameters and the ingest service has this new style with '.' delimited options following this pattern.  To get them both working currently the configuration just needs to repeat the values for each set of keys.